soweego/wikidata/sparql_queries.py

```diff
@@ -414,3 +414,43 @@ def query_birth_death(qids_bucket):
     query += """}"""
 
     return query
+
+
+def get_subclasses_of_qid(QID: str) -> Set[str]:
+    """
+    Given a QID, return the QID of it's subclasses
+
+    :param qid: QID we want to get the subclasses for
+
+    :return: set with QIDs of subclasses
+    """
+
+    # subclasses (?res is subclass of QID)
+    result = make_request("""
+                SELECT DISTINCT ?res WHERE {
```

Comment:
Please convert the queries to constants and move them to the top of the file.
Please also remember to replace the hardcoded binding `?res` with constants as well.

Reply:
good idea, done in 1a3d871